<launch>
  <arg name="robot" default="spot"/>

  <node pkg="sloop_ros" type="sloop_main"
        name="sloop_main" ns="$(arg robot)" output="screen">
  </node>

  <node pkg="sloop_ros" type="sloop_belief"
        name="sloop_belief" ns="$(arg robot)" output="screen">
  </node>

  <node pkg="sloop_ros" type="sloop_planning"
        name="sloop_planning" ns="$(arg robot)" output="screen">
    <param name="belief_topic" value="belief"/>
  </node>
</launch>
